src/server/frontend/render.js

```diff
@@ -72,15 +73,16 @@ function fetchComponentData(dispatch, req, {components, location, params}) {
   });
 }
 
-function renderPage(store, renderProps, req) {
+async function renderPage(store, renderProps, req) { // eslint-disable-line space-before-function-paren
   const clientState = store.getState();
   const {headers, hostname} = req;
   const appHtml = getAppHtml(store, renderProps);
-  const scriptHtml = getScriptHtml(clientState, headers, hostname);
+  const {js: appJsFilename, css: appCssFilename} = await getAppAssetFilenamesAsync();
```

Comment:
I will review it tomorrow since sunday is time for... pet projects ;) Feel free to refactor or whatever it before that anytime. Thank you very much!